### ISSUE TITLE
PyDocStyleParser fails with IndexOutOfBoundsException

### ISSUE THREAD

**Issue by uhafner** [tomasbjerre/violations-lib#71]
The PyDocStyleParser has a problem with some files: 
```
IndexOutOfBoundsException: Index: 0, Size: 0
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:657)
	at java.util.ArrayList.get(ArrayList.java:433)
	at se.bjurr.violations.lib.parsers.PyDocStyleParser.parseReportOutput(PyDocStyleParser.java:26)
	at 
```

Should be part of violations-lib 1.93.
See related issue in Jenkins Warnings plugin: https://issues.jenkins-ci.org/browse/JENKINS-57799. I already pinged the reporter of the Jenkins issue to provide an example file...

**Comment by davidhart82** [comment]
This occurs if you specify `-s` to pydocstyle, i.e. show source.  The parser is not expecting source in the output.

**Comment by uhafner** [comment]
@davidhart82 Maybe it would help if you can add a snippet with one of these warning logs that contains the source.

**Comment by davidhart82** [comment]
Sure, a simple test file, without the `-s` flag produces errors like so:

```
test.py:1 at module level:
        D100: Missing docstring in public module
test.py:1 in public function `my_test_function`:
        D103: Missing docstring in public function
```

With the `-s` flag, the same errors look like:

```
test.py:1 at module level:
        D100: Missing docstring in public module

 1: d 2: e 3: f 4:   5: m 6: y 7: _        ...


test.py:1 in public function `my_test_function`:
        D103: Missing docstring in public function

 1: d 2: e 3: f 4:   5: m 6: y 7: _        ...


```

**Comment by tomasbjerre** [comment]
Released a fix in 1.106, released to JCenter.

Would be nice to have more extensive example reports to test the parser with.
